data/plugins/language_lua.lua

```diff
@@ -52,9 +96,9 @@ syntax.add {
     ["or"]       = "keyword",
     ["goto"]     = "keyword",
     ["self"]     = "keyword2",
-    ["true"]     = "literal",
-    ["false"]    = "literal",
-    ["nil"]      = "literal",
+    ["true"]     = "number",
+    ["false"]    = "number",
+    ["nil"]      = "number",
```

Comment:
Why was this changed? Other languages use `literal` for those keywords.

Reply:
as I mentioned on discord, because all the color schemes symbol token is set to the same color as normal leaving me with poor choices, and wanted to differentiate true, false, nil from sub fields, but I can revert if for others it is annoying